developer/contributing.rst

```diff
@@ -32,16 +32,20 @@ Setup
 Once you have chosen an issue to work on, read the ``README`` of the
 repository in which the issue has been opened and follow the setup
 instructions, each module has its own specific instructions and we highly
-suggest you to read as much as possible.
+suggest you to read as much as possible. If there are errors try asking via our
+communication channels, as a last resort open an issue or try and fix it and create a pull
+request to fix it.
 
 How to commit your changes properly
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Our main development branch is master, it's our central development
 branch.
 
-Your code should be pushed from a different branch and then it is merged
-into master after proper review from the project maintainers.
+Your code should be pushed from your fork and after a Pull Request
+and proper reviews from the project maintainers and testing from 
+travis/coveralls then it will be merged into master. Please be patient
+for reviews from the project maintainers, they are extremely busy people.
```

Comment:
please remove "Please be patient for reviews from the project maintainers, they are extremely busy people."

Reply:
Please change the remaining passage as follows:

```Your code should be pushed from your fork and when ready
you should open a pull request on github. The pull request will
be merged only once the test build completes successfully
(automated tests, code coverage check, style checks, etc.)
and after project maintainers have reviewed and tested it.```